### DISCUSSION TITLE
regolith-session-sway missing on jammy?

### DISCUSSION THREAD

**Discussion by gdevenyi** [regolith-linux/regolith-desktop#918] [Q&A]
All the install guides refer to regolith-session-sway for wayland support, however on Ubuntu Jammy this package doesn't exist. Is there some other restriction I'm missing?

**Comment by gdevenyi** [comment]
The answer was that the repository URL has been changed and that doesn't seem to be mentioned in the migration guide.

**Reply by kgilmer** [reply]
@gdevenyi , just checking but is there somewhere on regolith-desktop.com that you're seeing inconsistent repository URLs?

**Reply by gdevenyi** [reply]
No, I had previously installed the repository and not installed regolith. The only inconsistent part is I didn't see any explicit note that the repo URL had been changed for the new release.
